components/ViewHabits.tsx

```diff
@@ -1,12 +1,34 @@
 import { css } from '@emotion/react';
 import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import Habit from './Habit';
+import { Habit as HabitType } from '../data-types';
 
-type Props = {};
+const fetchData = async () => {
+  const data = await fetch('../data/habits.json');
+  const json = await data.json();
+  return json;
+};
+
+const ViewHabits = () => {
+  const [habitsData, setHabitsData] = useState<HabitType[]>([]);
+  useEffect(() => {
+    const fetch = async () => {
+      const data = await fetchData();
+      setHabitsData(data);
+    };
+    fetch();
+  }, []);
 
-const ViewHabits = (props: Props) => {
   return (
     <div css={containerStyle}>
-      <Image src='/logo.svg' alt='.SHIFT logo' fill />
+      <>
```

Comment:
[curious] is there a reason to use a Fragment here? Shouldn't need to, considering that multiple children are permissable